Tasks/VsTest/runvstest.ts

```diff
@@ -6,38 +6,46 @@ import * as path from 'path';
 import * as distributedTest from './distributedtest';
 import * as ci from './cieventlogger';
 import * as utils from './helpers';
+import * as os from 'os';
+const osPlat: string = os.platform();
 
-//Starting the VsTest execution
-const taskProps = { state: 'started', result: '' };
-ci.publishEvent(taskProps);
+tl.setResourcePath(path.join(__dirname, 'task.json'));
 
-try {
-    tl.setResourcePath(path.join(__dirname, 'task.json'));
-    utils.Helper.setConsoleCodePage();
-    const useDtaExecutionEngine = isDtaEngineRequired();
-    if (useDtaExecutionEngine) {
-        ci.publishEvent({
-            runmode: 'distributedtest', parallelism: tl.getVariable('System.ParallelExecutionType'),
-            testtype: tl.getInput('testSelector')
-        });
+if (osPlat !== 'win32') {
```

Comment:
did you evaluate "platforms" option in execution?

Reply:
i set the platform to linux and it still ran on a windows machine

Reply:
Will file a bug on them